ClusterTest/src/main/scala/net/snowflake/spark/snowflake/testsuite/StressReadWriteSuite.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015-2020 Snowflake Computing
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.snowflake.spark.snowflake.testsuite
+
+import net.snowflake.client.jdbc.internal.fasterxml.jackson.databind.ObjectMapper
+import net.snowflake.spark.snowflake.{ClusterTestResultBuilder, DefaultJDBCWrapper, TestUtils}
+import org.apache.spark.sql.SparkSession
+
+import scala.collection.JavaConverters.asScalaBufferConverter
+import scala.io.Source
+
+/**
+  * Similar to BasicReadWriteSuite, but configurable with a JSON file as to which tables
+  * to perform the read and write on.
+  */
+class StressReadWriteSuite extends ClusterTestSuiteBase {
+
+  // A source table to read from for a stress test is defined by its schema and table name
+  case class StressTestSourceTable(schema: String,
+                                   tableName: String)
+
+  /**
+    * Reads the json file as configured by the system env variable defined in TestUtils.STRESS_TEST_SOURCES
+    * to get the tables to perform the read/write stress test on.
+    *
+    * The file should be in the format:
+    *
+    * [
+    *   {
+    *     "schema": "table1_schema",
```

Comment:
Nice to have: it should be better to add "database" for the json file to give the flexibility.